internal/bundle/bundles/python_wheel_task/template/databricks.yml.tmpl

```diff
@@ -14,6 +19,7 @@ resources:
             num_workers: 1
             spark_version: "{{.spark_version}}"
             node_type_id: "{{.node_type_id}}"
+            data_security_mode: USER_ISOLATION
```

Comment:
Curious, why was this needed?

Reply:
@pietern wheels from WSFS only work for 13.1+ Shared or 13.2+ Assigned clusters, but do not work on No Isolation clusters at all (which it was before the change)

Reply:
Of course, thanks.